bvm/ballerina-core/src/main/java/org/ballerinalang/bre/bvm/CPU.java

```diff
@@ -3637,6 +3637,16 @@ private static void calculateLength(WorkerExecutionContext ctx, int[] operands,
         if (typeTag == TypeTags.XML_TAG) {
             sf.longRegs[j] = ((BXML) entity).length();
             return;
+        } else if (typeTag == TypeTags.TABLE_TAG) {
+            BTable bTable = (BTable) entity;
+            // Check if the table reference is null, if so throw a null reference error
```

Comment:
Better to end comments with a full stop.

Reply:
Added full stops to comments with 96af83f